ontology/rml-core.owl

```diff
@@ -0,0 +1,517 @@
+      rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+      rdfs:label "Join" .
+
+
+###  http://w3id.org/rml/LanguageMap
+:LanguageMap rdf:type owl:Class ;
+             rdfs:subClassOf :ExpressionMap ;
+             rdfs:comment "Represents a language map" ;
+             rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+             rdfs:label "Language Map" .
+
+
+###  http://w3id.org/rml/Literal
+:Literal rdf:type owl:Class ;
+         rdfs:subClassOf owl:Thing ;
+         rdfs:comment "Denotes a Literal, used with termType."@en ;
+         rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+         rdfs:label "Literal" .
+
+
+###  http://w3id.org/rml/LogicalSource
+:LogicalSource rdf:type owl:Class ;
+               rdfs:comment "Represents a logical source." ;
+               rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+               rdfs:label "Logical Source" .
+
+
+###  http://w3id.org/rml/LogicalTarget
+:LogicalTarget rdf:type owl:Class ;
+               rdfs:comment "Represents a Logical Target" ;
+               rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+               rdfs:label "Logical Target" .
+
+
+###  http://w3id.org/rml/ObjectMap
+:ObjectMap rdf:type owl:Class ;
+           rdfs:subClassOf :TermMap ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :datatype ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Datatype
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :object ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Resource
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :object ;
+                             owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Resource
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :language ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onDataRange xsd:string
+                           ] ;
+           rdfs:comment "Represents an object map."@en ;
+           rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+           rdfs:label "Object Map" .
+
+
+###  http://w3id.org/rml/PredicateMap
+:PredicateMap rdf:type owl:Class ;
+              rdfs:subClassOf :TermMap ,
+                              [ rdf:type owl:Restriction ;
+                                owl:onProperty :predicate ;
+                                owl:qualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                                owl:onClass rdf:Property
+                              ] ;
```

Comment:
This belongs on `:PredicateObjectMap`. And I don't see why we should restrict the cardinality there. So this should not be bound IMO.